core/src/main/java/org/mskcc/cbio/maf/MafUtil.java

```diff
@@ -969,4 +977,47 @@ else if (!refAllele.equalsIgnoreCase(record.getTumorSeqAllele2()))
 
 		return key;
 	}
+    
+    /**
+     * Resolve tumor seq allele given a reference allele, tumor seq allele1, and tumor seq allele2. 
+     * SNPs will be preferred over INDELs in cases where tumor seq allele1 and tumor seq allele2 contain
+     * both a SNP and '-'.
+     * @param referenceAllele
+     * @param tumorSeqAllele1
+     * @param tumorSeqAllele2
+     * @return 
+     * 
+     * @author angelicaochoa
+     */
+    public static String resolveTumorSeqAllele(String referenceAllele, String tumorSeqAllele1, String tumorSeqAllele2) {        
```

Comment:
please add unit tests with the different scenarios you would like to cover here.

Reply:
e.g. workout what will happen to these example scenarios:
* `ref = AACG, all1= AACG, all2 = - `
* `ref = AACG, all1= -, all2 = AACG `
* `ref = - , all1 = ATGC, all2 = - `
etc